modules/atomic-widgets/assets/js/editor/commands/styles.js

```diff
@@ -91,38 +96,39 @@ export class Styles extends $e.modules.editor.document.CommandHistoryDebounceBas
 
 	apply( args ) {
 		const { container, bind, meta, props } = args;
-		let styleDefId = args.styleDefId ?? null;
+		let styleDefID = args.styleDefID ?? null;
+
+		const oldStyles = structuredClone( container.model.get( 'styles' ) ) ?? {};
 
-		const oldStyles = container.model.get( 'styles' ) ?? {};
 		const oldBindSetting = container.settings.get( bind );
 		let style = {};
 
-		if ( ! styleDefId ) {
+		if ( ! styleDefID ) {
 			style = $e.internal( 'document/atomic-widgets/create-style', {
 				container,
 				bind,
 			} );
 
-			styleDefId = style.id;
+			styleDefID = style.id;
 		} else {
-			if ( ! oldStyles[ styleDefId ] ) {
+			if ( ! oldStyles[ styleDefID ] ) {
 				throw new Error( 'Style Def not found' );
 			}
 
-			style = oldStyles[ styleDefId ];
+			style = oldStyles[ styleDefID ];
 		}
 
 		if ( ! this.variantExists( style, meta ) ) {
 			$e.internal( 'document/atomic-widgets/create-variant', {
 				container,
-				styleDefId,
+				styleDefID,
 				meta,
 			} );
 		}
```

Comment:
if ( ! styleDefID ) {
			style = $e.internal( 'document/atomic-widgets/create-style', {
				container,
				bind,
			} );

			styleDefID = style.id;
		} else if ( oldStyles[ styleDefID ] ) {
		    style = oldStyles[ styleDefID ];
		} else {
			throw new Error( 'Style Def not found' );
		}